doc/index.rst

```diff
@@ -34,9 +34,27 @@ There are three different ways which you can use Cube Browser:
 Installation
 ____________
 
-Conda is great
+In the following pages you will find downloads of Jupyter notebooks containing Cube Browser operations.
+To be able to use these notebooks, you will need to configure your environment correctly.
+The most efficient and clean method to do this is to use a Conda environment (please see Anaconda website for help and information).
```

Comment:
I wonder if you can safely lose the bit in brackets as you say the same thing in the next paragraph, where you actually supply the link that you hint at in these brackets.